src/specialOffer/SpecialOffer.style.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components'
+
+import { space } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
+
+export const StyledSpecialOffer = styled.div``
+
+export const StyledSpecialOfferPicture = styled.img`
+  width: 70%;
```

Comment:
why?! hehehe

Reply:
It's approximately the ratio I see on the [Figma](https://www.figma.com/file/zX4OiDroAATUOJ4yl4ZEuO/Pixarization-final?node-id=1763%3A0). 

Why it the best way to handle this value?

Reply:
The percentage-based value will look bad on very large screens.
Should we combine it with a max-width: Xpx (maybe 400)?

Reply:
I've used 

```css
  width: 70%;
  max-width: 400px;
```

let me know if it's OK for you